backend/schemas/user.py

```
@@ -1,8 +1,62 @@
 from pydantic import BaseModel
+from typing import List, Optional
```

Ruff (0.8.2)

`typing.List` is deprecated, use `list` instead

(UP035)